data/brands/amenity/recycling.json

```
@@ -202,6 +202,19 @@
         "brand:ru": "Экобокс",
         "name": "Экобокс"
       }
+    },
+    {
+      "displayName": "TRAID",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "amenity": "recycling",
+        "brand": "TRAID",
+        "brand:wikidata": "Q66581837",
```

It looks like this is the wikidata page for [Saint Vincent de Paul thrift stores](https://www.wikidata.org/wiki/Q66581837).
```suggestion
        "brand:wikidata": "Q109717195",
```